cyhy/mailer/CyhyNotificationMessage.py

```diff
@@ -40,15 +40,15 @@ class CyhyNotificationMessage(ReportMessage):
 Cybersecurity and Infrastructure Security Agency (CISA)
 ncats@hq.dhs.gov
 
-WARNING: This document is FOR OFFICIAL USE ONLY (FOUO). It contains information that may be exempt from public release under the Freedom of Information Act (5 U.S.G. 552). It is to be controlled, stored, handled, transmitted, distributed, and disposed of in accordance with CISA policy relating to FOUO information and is not to be released to the public or other personnel who do not have a valid 'need-to-know' without prior approval of an authorized CISA official.
+WARNING: This message and any attached document(s) is FOR OFFICIAL USE ONLY (FOUO). It contains information that may be exempt from public release under the Freedom of Information Act (5 U.S.G. 552). It is to be controlled, stored, handled, transmitted, distributed, and disposed of in accordance with DHS policy relating to FOUO information and is not to be released to the public or other personnel who do not have a valid "need-to-know" without prior approval of an authorized DHS official.
```

Comment:
I suggest use the fancy HTML quotes around `need-to-know`.  See [here](https://practicaltypography.com/straight-and-curly-quotes.html), for instance.  I'm talking about `&ldquo;` and `&rdquo;`.

Reply:
> I suggest use the fancy HTML quotes around `need-to-know`. See [here](https://practicaltypography.com/straight-and-curly-quotes.html), for instance. I'm talking about `&ldquo;` and `&rdquo;`.

Thanks @jsf9k - see https://github.com/cisagov/cyhy-mailer/pull/70/commits/583f2e35679ce1d1441521b96d6160ebb6bd4e90